src/main/java/com/denizenscript/depenizen/bukkit/Depenizen.java

```diff
@@ -62,6 +64,12 @@ public void checkLoadBungeeBridge() {
         Debug.log("Loaded bungee bridge!");
     }
 
+    public void checkLoadClientizenBridge() {
+        if (getConfig().getBoolean("Clientizen.enabled")) {
```

Comment:
`, false`?

Reply:
```java
    public boolean getBoolean(@NotNull String path) {
        Object def = this.getDefault(path);
        return this.getBoolean(path, def instanceof Boolean ? (Boolean)def : false);
    }
```
That's Spigot's default (and is explicitly mentioned in the Javadocs).